src/app/requirements.txt

```diff
@@ -1,3 +1,3 @@
 biopython
 streamlit-aggrid
-git+https://github.com/aniketfadia96/neglected-diagnostics.git@seq-var-table
\ No newline at end of file
+git+https://github.com/aniketfadia96/neglected-diagnostics.git@assay-target
```

Comment:
Yes, this is a branch name. I have added a branch name so that I can host it on community cloud for Misa to try this feature. More details [here](https://docs.streamlit.io/streamlit-community-cloud/deploy-your-app).